File: qe_selenuim_python/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

driver = webdriver.Chrome()

driver.get("https://training-support.net")

print("Home Page Title:", driver.title)


about_us = driver.find_element(By.LINK_TEXT, "About Us")
about_us.click()

time.sleep(2)

print("About Us Page Title:", driver.title)

driver.quit()

File: qe_selenium_java/Activity1.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity1 {

    public static void main(String[] args) throws InterruptedException {

        WebDriver driver = new ChromeDriver();

        driver.get("https://training-support.net");

        System.out.println("Home Page Title: " + driver.getTitle());

        WebElement aboutUs = driver.findElement(By.linkText("About Us"));
        aboutUs.click();

        Thread.sleep(2000);
        System.out.println("About Us Page Title: " + driver.getTitle());

        driver.quit();
    }
}
